lms/static/my.js

```diff
@@ -106,14 +106,38 @@ function trackReadAllNotificationsButton(button) {
   });
 }
 
+function getPostUploadMessage() {
+  const myDropzone = Dropzone.forElement("#demo-upload");
+  const feedbacks = document.getElementById('upload-feedbacks');
+  const matchesSpan = document.getElementById('upload-matches');
+  const missesSpan = document.getElementById('upload-misses');
+  myDropzone.on('success', function() {
+    const args = Array.prototype.slice.call(arguments)[1];
+    const matches = args['exercise_matches'];
+    const misses = args['exercise_misses'];
+    if (matches.length) {
+      matchesSpan.append(matches + ',');
+    }
+    if (misses.length) {
+      missesSpan.append(misses + ',');
+    }
+    feedbacks.classList.remove('d-none');
+  });
+}
+
 window.escapeUnicode = escapeUnicode;
 
 window.addEventListener('load', () => {
-  const codeElement = document.getElementById('code-view').dataset;
-  const solutionId = codeElement.id;
   updateNotificationsBadge();
   trackReadAllNotificationsButton(document.getElementById('read-notifications'));
-  trackCopyButton(document.getElementById('copy-button'), document.getElementById('user-code').textContent);
-  trackShareSolution(solutionId, document.getElementById('share-action'));
-  trackDisableShareButton(solutionId, document.getElementById('cancel-share'));
+  try {
+    const codeElement = document.getElementById('code-view').dataset;
+    const solutionId = codeElement.id;
+    trackCopyButton(document.getElementById('copy-button'), document.getElementById('user-code').textContent);
+    trackShareSolution(solutionId, document.getElementById('share-action'));
+    trackDisableShareButton(solutionId, document.getElementById('cancel-share'));
+  } catch(e) {}
+  try {
+    getPostUploadMessage();
```

Comment:
This will load dropzone in each page we don't have code-view.

Use `if`'s instead of `try` to split the code so each segment will load only on the correct page.

`if (document.getElementById('code-view') !== undefined) { ... }`